MRML/vtkIGTLToMRMLImage.cxx

```diff
@@ -256,303 +219,67 @@ int vtkIGTLToMRMLImage::IGTLToMRML(igtl::MessageBase::Pointer buffer, vtkMRMLNod
     return 0;
     }
 
-  // Create a message buffer to receive image data
-  igtl::ImageMessage::Pointer imgMsg;
-  imgMsg = igtl::ImageMessage::New();
-  imgMsg->Copy(buffer); // !! TODO: copy makes performance issue.
+  if (!Codec.Get())
```

Comment:
Minor coding convention comment: typically local variables in functions start with a lower case letter, and member variables start with an uppercase.